### PR TITLE
warp7: update boot partition

### PR DESCRIPTION
**Pull request by texierp**
Replace fatload command and location (zImage, dtb).
This is a requirement for mender (partition layout)):

> - one boot partition, containing the U-Boot bootloader and its environment
> - two partitions for storing the root file system and kernel. The kernel image file, zImage, and any device tree binary should be stored in directory /boot

--
Pierre-Jean

### CODE REVIEW

File: include/configs/warp7.h [hunk 1]
```diff
@@ -45,11 +45,11 @@
 	"mmcargs=setenv bootargs console=${console},${baudrate} " \
 		"root=PARTUUID=${uuid} rootwait rw\0" \
 	"loadbootscript=" \
-		"fatload mmc ${mmcdev}:${mmcpart} ${loadaddr} ${script};\0" \
+		"load mmc ${mmcdev}:1 ${loadaddr} ${script};\0" \
```
Comment (otavio) [inline include/configs/warp7.h:48]: You did change the mmcpart here and this is not part of this commit.

File: include/configs/warp7.h [hunk 2]
```diff
@@ -22,20 +22,20 @@
...
+#define CONFIG_SYS_MMC_IMG_LOAD_PART	2
 
 #define CONFIG_DFU_ENV_SETTINGS \
 	"dfu_alt_info=boot raw 0x2 0x400 mmcpart 1\0" \
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	CONFIG_DFU_ENV_SETTINGS \
 	"script=boot.scr\0" \
-	"image=zImage\0" \
+	"image=/boot/zImage\0" \
```
Comment (otavio) [inline include/configs/warp7.h:33]: This change the way we use the images. 

I understand why you are doing that but it seems like you will need to have a patch for this. However, we can reduce the changes need using a bootdir variable or similar so you reduce the conflict risk.

**Review by otavio — CHANGES_REQUESTED** [review]

File: include/configs/warp7.h (hunk 1 above)
Comment (texierp) [reply include/configs/warp7.h:48]: Yes sorry for that.

File: include/configs/warp7.h (hunk 2 above)
Comment (texierp) [reply include/configs/warp7.h:33]: Thanks Otavio I understand, the best way to handle this integration is to work directly into the distro layer (https://github.com/WaRP7/meta-warp7-distro ), I think ? and keep this configuration ;)

Sorry for the noise ;)

---
Pierre-Jean
